packages/turf-rectangle-grid/test/in/victoria-20x100-km.json

```diff
@@ -1,5 +1,5 @@
 {
-  "bbox": [141, -34, 150, -39],
+  "bbox": [141, -39, 150, -34],
```

Comment:
Did you flip this because the behavior is changing? Or to alter the test?

Reply:
Technically the previous bbox was incorrect because the miny is supposed to come first